components/docs-chef-io/content/automate/chef_automate_profile.md

```diff
@@ -0,0 +1,55 @@
++++
+title = "User Profile"
+
+draft = false
+
+gh_repo = "automate"
+[menu]
+  [menu.automate]
+    title = "User Profile"
+    parent = "automate/settings"
+    identifier = "automate/settings/chef_automate_profile.md User Profile"
+    weight = 120
++++
+
+## Overview
+
+Chef Automate users can manage their own display name, date/time format, and password. Select the user icon in the top navigation bar, and then select **Profile** from the drop-down menu.
+
+{{< figure src="/images/automate/chef-automate-profile.png" alt="Profile Section in Chef Automate">}}
+
+Select **Profile** to view the details with respect to your _username_. The **Details** tab lets you update your _Display Name_ and the _Time Format_, whereas the **Reset Password** tab lets you update your password.
+
+### Change a Display Name
+
+You can change the display name for your Chef Automate in the bar.
+
+{{< figure src="/images/automate/display-name-in-profile.png" alt="Display Name of Chef Automate">}}
+
+{{< note >}}
+The **Display Name** gets updated as soon as you change the name in the bar.
+{{< /note >}}
+
+The Display Name bar should not be kept empty or else it throws error.
+
+{{< figure src="/images/automate/error-in-display-name-bar.png" alt="Error in Display Name">}}
+
+### Change the Date/Time Format
+
+You can select one of five date and time formats.
+
+{{< figure src="/images/automate/date-time-format-in-profile.png" alt="Date/Time Format">}}
+
+Select the date/time format and save to get the changes reflected in the interface.
+
+{{< note >}}
+The **Date/Time Format** gets updated as soon as you select a new time format from the drop down.
```

Comment:
Again, wouldn't the user need to `save` first?

Reply:
Yes!
Have updated the same